test/representers/api/story_representer_test.rb

```diff
@@ -24,6 +24,15 @@
       d_story.title.must_equal 'title'
       d_story.account_id.must_equal 8
     end
+
+    it 'can set the series' do
```

Comment:
What's this test about?  Just making sure you can move between series you have account access to?

Reply:
Yes - I wrote it when I was trying to debug issues with setting the series via the controller - the issue was ultimately not a problem with the representer, and this was already working fine. I can delete the test if you like! goodness knows this suite doesn't need to be bigger.

Reply:
Nah, that makes sense.  What's 1 more test to this gigantic test suite?